blueocean-pipeline-editor/src/main/js/EditorPage.jsx

```diff
@@ -333,9 +345,24 @@ class PipelineLoader extends React.Component {
                 }
                 const pipelineScript = Base64.decode(content.base64Data);
                 this.setState({ sha: content.sha });
+
                 convertPipelineToJson(pipelineScript, (p, err) => {
                     if (!err) {
                         const internal = convertJsonToInternalModel(p);
+
+                        //check if pipeline contains sequential parallel stages and show error if it does
+                        this.checkForNestedSequentialStages(internal, errParentStageName => {
```

Comment:
We should come up with generic code at this point to handle incompatibility. do not want to hold this PR, something we should do in another PR.